modules/serverless-create-domain-mapping-kn.adoc

```diff
@@ -30,7 +30,7 @@ $ kn domain create <domain_mapping_name> --ref <target_name>
 .Example command
 [source,terminal]
 ----
-$ kn domain create example-domain-map --ref example-service
+$ kn domain create example.domain --ref example-service
```

Comment:
Super small nit:
`example.com` or `example.org` are reserved by IANA for the general purpose such as documentation so we might have better to use them rather than `example.domain`.

ref: https://en.wikipedia.org/wiki/Example.com

Reply:
Sounds good, thank you!